index.html

```diff
@@ -0,0 +1,47 @@
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <h1>Hello World, we are software developers!
+    </h1>
+    <main id="main">
+      <h1 id="title">Nelson Mandela</h1>
+      <div id="img-div">
+        <img src="image/nelsonmandellalogo.jpg" id="image" alt="Nelson Mandella Image">
+        <p id=img-caption>  He is held in deep respect within South Africa, where he is often referred to by his Thembu clan name, Madiba, and described as the "Father of the Nation".</p>
+         </div>
+        <div id="tribute-info">
+          <ul>
+            <li> Nelson Rolihlahla Mandela (/mænˈdɛlə/;[1] Xhosa: [xolíɬaɬa mandɛ̂ːla]; 18 July 1918 – 5 December 2013) was a South African anti-apartheid revolutionary and political leader who served as the first president of South Africa from 1994 to 1999. He was the country's first black head of state and the first elected in a fully representative democratic election. His government focused on dismantling the legacy of apartheid by tackling institutionalised racism and fostering racial reconciliation. Ideologically an African nationalist and socialist, he served as the president of the African National Congress (ANC) party from 1991 to 1997.</li>
+            <li>A Xhosa, Mandela was born into the Thembu royal family in Mvezo, Union of South Africa</li>
+            <li>He studied law at the University of Fort Hare and the University of Witwatersrand before working as a lawyer in Johannesburg..</li>
+            <li> There he became involved in anti-colonial and African nationalist politics, joining the ANC in 1943 and co-founding its Youth League in 1944</li>
+            <li>After the National Party's white-only government established apartheid, a system of racial segregation that privileged whites, Mandela and the ANC committed themselves to its overthrow</li>
+            <li> He was appointed president of the ANC's Transvaal branch, rising to prominence for his involvement in the 1952 Defiance Campaign and the 1955 Congress of the People</li>  
+          </ul>      
+      </div>
```

Comment:
Proper indentation Should be Provided to opening and closing tags. as it helps in debugging